recipes/sip/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+    pythonw configure.py --sysroot=$PREFIX
```

Comment:
It's used for any GUI programs on Mac. See this [ref]( https://docs.python.org/3/using/mac.html#running-scripts-with-a-gui ).